### DISCUSSION TITLE
Zombie process with no threads

### DISCUSSION THREAD

**Discussion by TETYYS** [winsiderss/systeminformer#1505] [Questions / Answers] [answered]
I have a zombie process that is indefinetely in exiting state with no threads alive which is being blocked by a faulty driver. Now, I know that it exits once I close my xbox 360 controller emulation software, but is there a way to find what holds it up using System Informer?

![image](https://user-images.githubusercontent.com/3091645/203387877-e39d2192-35bd-42fa-b674-c5075c400d83.png)

Only memory statistics (76 kB private bytes and some others) and Token tab is populated, everything else is either 0 or empty.

**Comment by dmex** [comment, marked as answer]
> is there a way to find what holds it up using System Informer?

The process is too far gone for SI to show anything.

> zombie process that is indefinetely in exiting state with no threads alive

 You need to dump kernel objects and check references using a kernel debugger. You can create a live kernel dump using the SI main menu > Tools > `Create live dump` option and open the dump file in windbg/windbg preview?

**Reply by TETYYS** [reply]
Found x360ce.exe with
```
!process 6f3c
!findhandle ffffc10670519080
```
Thanks
